data/2018-11/agile-tour-bangkok-2018.md

```diff
@@ -0,0 +1,27 @@
+---
+id: gucode-is-cool-201811
+date: "2018-11-02"
+time: "09:00 ~ 17:00"
+location:
+  title: TODO
+  url: https://www.google.com/maps/place/
+categories:
+  - Conference
+topics:
+  - Agile
+links:
+  - type: website
+    url: https://www.facebook.com/events/1430324887111024/
+    title: Facebook Event
+  - type: ticket
+    url: https://l.facebook.com/l.php?u=http%3A%2F%2Fatb2018.eventbrite.com%2F&h=AT060oXOaPFkuQ_4mXcQ-43GrSIqkzLIMjUrZLCyGEshSD-m7kcPNnSSAJPa9VsMcnyAXqSxpJq6zkecJ2qOjwQxs2-wvz6-vKFRjvHoZXBJvALGKk-6wSNDu6lQW-VEfCMRM13V
```

Comment:
This looks like a Facebook redirect link. Can you update to actual Eventbrite link?